internal/api/api.go

```diff
@@ -50,6 +52,6 @@ func (api *API) registerRoutes() {
 
 func (api *API) Start(address string) error {
 	api.registerRoutes()
-	api.logger.Debug("Starting API on ", "address", address)
+	api.logger.Debug(address, "Starting API on ")
```

Comment:
This should be `api.logger.Debug("startup", "Starting API", "address", address)`